packages/templates/src/raw/doers-get-more-done/template.json

```diff
@@ -220,7 +220,7 @@
           "focalY": 50,
           "basedOn": "4e256c35-5e30-45f5-9095-d8ce0a230b21",
           "id": "8e053a88-17a5-4162-8ae0-214e5a84e33c",
-          "content": "<span style=\"color: #fafae4\">EXPAND YOUR</span>\n<span style=\"color: #fafae4\">LIVING SPACE TO</span>",
+          "content": "<span style=\"color: #fafae4; text-transform: uppercase\">Expand Your</span>\n<span style=\"color: #fafae4; text-transform: uppercase\">Living Space to</span>",
```

Comment:
[Suggestion Only] Can go with `Expand your living space to` else only the last word gets left out from using Title case